<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VidSel</class>
 <widget class="QWidget" name="VidSel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Devices</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin" stdset="0">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin" stdset="0">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Select Video Device:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="devlist">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="_3">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin" stdset="0">
        <number>0</number>
       </property>
       <item>
        <widget class="vidlab" name="video_label">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>120</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="icons.qrc">:/new/prefix1/greenguy.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="flip">
         <property name="text">
          <string>Flip video</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="face_is_blue">
         <property name="text">
          <string>My face looks blue</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_4">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin" stdset="0">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="ok_button">
           <property name="text">
            <string>&amp;Done</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="use_vfw">
     <property name="text">
      <string>Attempt to use VFW drivers (for older cameras.)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="_5">
     <item>
      <widget class="QPushButton" name="source_button">
       <property name="text">
        <string>Source Brightness/Colors/etc.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="format_button">
       <property name="text">
        <string>Video Format Selections</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>vidlab</class>
   <extends>QLabel</extends>
   <header>vidlab.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
